tests/fips/openssl/openssl_pubkey_dsa.pm

```diff
@@ -52,7 +52,7 @@ sub run {
         assert_script_run "openssl dsa -in $dsa_prikey -pubout -out $dsa_pubkey";
 
         # A source of random numbers is required for DSA, so get message digest first
-        assert_script_run "openssl dgst -$dgst_alg $file_raw | awk '{print $2}' > $file_dgt";
+        assert_script_run sprintf('openssl dgst -%s %s | awk \'{print $2}\' > %s', $dgst_alg, $file_raw, $file_dgt);
```

Comment:
maybe like this: `sprintf("openssl dgst -%s %s | awk \"{print $2}\" > %s", $dgst_alg, $file_raw, $file_dgt);`

Reply:
What about even this?
```suggestion
        assert_script_run sprintf(q[openssl dgst -%s %s | awk '{print $2}' > %s], $dgst_alg, $file_raw, $file_dgt);
```

Reply:
Both are still the same to me. :)